R/hugo.R

```diff
@@ -29,6 +29,7 @@ hugo_build = function(local = FALSE) {
   config = load_config()
   # Hugo 0.48 generates an ugly empty resources/ dir in the root dir
   on.exit(bookdown:::clean_empty_dir('resources'), add = TRUE)
+  on.exit(run_script('R/after_build.R', as.character(local)), add = TRUE)
```

Comment:
`hugo_build()` seems to be the wrong place to add this. I think it should be added to `build_site()` instead after this line: https://github.com/rstudio/blogdown/blob/44895dde48d556fca24d788de407d0c5e868ecbd/R/render.R#L48

Reply:
Moved as requested.